pkg/sensors/tracing/kprobe_test.go

```diff
@@ -3884,6 +3884,18 @@ func TestKprobeMatchBinaries(t *testing.T) {
 		}
 		matchBinariesTest(t, "NotPrefix", []string{"/usr/bin/t"}, createBinariesChecker("/usr/bin/head", "/etc/passwd"))
 	})
+	t.Run("Postfix", func(t *testing.T) {
+		if !kernels.EnableLargeProgs() {
+			t.Skip(skipMatchBinariesPrefix)
```

Comment:
you can update the `skipMatchBinariesPrefix` const string to include Postfix/NotPostfix